Druid/DRBytecodeGenerator.class.st

```diff
@@ -121,6 +127,24 @@ DRBytecodeGenerator >> targetAST [
 	^ method
 ]
 
+{ #category : 'accessing' }
+DRBytecodeGenerator >> vectorTempIndex [
+
+	^ vectorTempIndex ifNil: [ ^ 0 ]
+]
+
+{ #category : 'accessing' }
+DRBytecodeGenerator >> vectorTempIndex: anInteger [
+
+	vectorTempIndex := anInteger
+]
+
+{ #category : 'accessing' }
+DRBytecodeGenerator >> vectorTempName [
+
+	^ '0vector0' ", self vectorTempIndex asString"
```

Comment:
why hardcoded?

Reply:
We are moving this to the CFG in the next iteration.